src/builder-module.c

```diff
@@ -1237,6 +1237,46 @@ shell (GFile          *app_dir,
 static const char skip_arg[] = "skip";
 static const char strv_arg[] = "strv";
 
+static gchar*
+get_libdir(GFile          *app_dir,
+           const char     *module_name,
+           BuilderContext *context,
+           GFile          *source_dir,
+           const char     *cwd_subdir,
+           char          **flatpak_opts,
+           const char     *prefix,
+           GError        **error) {
+  g_autofree char *output = NULL;
+  g_autoptr(GFile) cwd_file = NULL;
+  g_autoptr(GPtrArray) args =
+    setup_build_args (app_dir, module_name, context, source_dir, cwd_subdir, flatpak_opts, NULL, &cwd_file);
+
+  g_ptr_array_add (args, g_strdup ("gcc"));
+  g_ptr_array_add (args, g_strdup ("-print-multiarch"));
```

Comment:
I'm somewhat worried about relying on gcc features in the runtime for this. I wonder if we can instead store this info in the sdk somehow? We've been talking about other things like this, for instance a default CFLAGS for security flags.